queries/erlang/highlights.scm

```diff
@@ -26,79 +62,68 @@
 
 [
   "catch"
-	"try"
-	"throw"
+  "try"
 ] @exception
+((atom) @exception (#any-of? @exception "error" "throw" "exit"))
+
+((atom) @boolean (#any-of? @boolean "true" "false"))
+
+;; Macros
+(macro_call_expr
+  "?" @macro
+  name: (_) @macro
+) @macro
```

Comment:
```suggestion
) @constant.macro
```
Not quite clear what you're trying to achieve here, though -- isn't that the same as simply
```
(macro_call_expr) @macro
```